src/base/bittorrent/session.cpp

```diff
@@ -1780,7 +1781,7 @@ void Session::processShareLimits()
 {
     qDebug("Processing share limits...");
 
-    foreach (TorrentHandle *const torrent, m_torrents) {
+    for (TorrentHandle *const torrent : qAsConst(m_torrents)) {
```

Comment:
I found this too, `deleteTorrent()` modifies m_torrents, i'll change `m_torrents` to `torrents()`